ui/v2/src/components/scenes/ScenePlayer/ScenePlayer.tsx

```diff
@@ -135,7 +135,11 @@ export class ScenePlayerImpl extends React.Component<IScenePlayerProps, IScenePl
       };
 
       getCurrentTimeHook = (_videoTag: any) => {
-        return _videoTag.currentTime + _videoTag.start;
+        let start = _videoTag.start
+        if (!start) {
+          start = 0;
+        }
```

Comment:
`const start = _videoTag.start || 0` is functionally the same, or `const { start = 0 } = _videoTag` if you want to get fancy.

Reply:
Done.